src/test/java/com/klarna/hiverunner/examples/rule/HelloAnnotatedHiveRunnerRule.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (C) 2013-2019 Klarna AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.klarna.hiverunner.examples.rule;
+
+import com.google.common.collect.Sets;
+import com.klarna.hiverunner.HiveShell;
+import com.klarna.hiverunner.StandaloneHiveRunner;
+import com.klarna.hiverunner.annotations.HiveProperties;
+import com.klarna.hiverunner.annotations.HiveResource;
+import com.klarna.hiverunner.annotations.HiveRunnerSetup;
+import com.klarna.hiverunner.annotations.HiveSQL;
+import com.klarna.hiverunner.annotations.HiveSetupScript;
+import com.klarna.hiverunner.config.HiveRunnerConfig;
+import org.apache.commons.collections.MapUtils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Hive Runner Reference implementation.
+ * <p/>
+ * All HiveRunner tests should run with the StandaloneHiveRunner
+ */
+@RunWith(StandaloneHiveRunner.class)
+public class HelloAnnotatedHiveRunnerRule {
```

Comment:
This class is not a rule maybe the original name is best: `HelloAnnotatedHiveRunner` (the package is already different. Same for other class in this package

Reply:
Agree.